woocommerce/woocommerce-android
# Replace deprecated menu usage

Closes: #7436
Closes: #7437
Closes: #7438
Closes: #7439


### Description

Replaces deprecated menu usage in:
* ShippingCarrierRatesFragment
* ShippingCustomsFragment
* ShippingLabelCreateCustomPackageFragment
* ShippingLabelCreateServicePackageFragment

### Testing instructions

* Start label creation flow from the order
* Check that carrier rates, customs, and package creation (both custom and service) have the "done" menu and this button works
